pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -117,6 +128,20 @@ def _flatten(self):
     def _unflatten(cls, data, _) -> "HilbertSchmidt":
         return cls(*data)
 
+    @property
+    def resource_params(self) -> dict:
+        u_ops = self.hyperparameters["U"]
+        v_ops = self.hyperparameters["V"]
+        if isinstance(u_ops, Operator):
+            u_ops = [u_ops]
+        if isinstance(v_ops, Operator):
+            v_ops = [v_ops]
```

Comment:
Actually I'm looking at the `__init__` which already contains this logic. When is `u_ops` ever not an iterable?